prisma/schema.prisma

```diff
@@ -73,15 +73,17 @@ model Installation {
 }
 
 model Repository {
-  id             String   @id @default(cuid())
-  githubId       Int      @unique
-  name           String
-  description    String?
-  homepage       String?
-  topics         String[]
-  default_branch String?
-  installationId String
-  configured     Boolean  @default(false)
+  id                 String   @id @default(cuid())
+  githubId           Int      @unique
+  name               String
+  description        String?
+  homepage           String?
+  topics             String[]
+  default_branch     String?
+  installationId     String
+  configured         Boolean  @default(false)
+  logoHref           String?
```

Comment:
`logoHref` -> `logoUrl` since it's more common and not tied to a html element prop naming.